woocommerce-gateways-country-limiter.php

```diff
@@ -39,8 +39,8 @@ public function __construct() {
 
 
             $this->settings = get_option( 'woocommerce_gateways_country_limiter' );
-
-            add_action( 'init', array( $this, 'init' ), 10 );
+            //set priority to 100 to make sure all gateways are loaded
+            add_action( 'init', array( $this, 'init' ), 100 );

```

Comment:
@mihaigrigori 100 may seem arbitrary. What about using `PHP_INT_MAX`?

Reply:
yes PHP_INT_MAX should be also fine